PlayCover/Views/Sidebar Views/HomeView.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  HomeView.swift
+//  PlayCover
+//
+//  Created by Isaac Marovitz on 06/08/2022.
+//
+
+import SwiftUI
+
+struct HomeView: View {
+    @EnvironmentObject var appsVM: AppsVM
+
+    // swiftlint: disable line_length
+
+    var body: some View {
+        ZStack {
+            ScrollView {
+                VStack(alignment: .leading) {
+                    LargeAppView(app: appsVM.apps[0])
+                    Spacer()
+                        .frame(height: 20)
+                    HStack {
+                        HalfAppBannerView(app: appsVM.apps[0], gradient: LinearGradient(colors: [Color(hue: 50/360, saturation: 1, brightness: 0.8), Color(hue: 10/360, saturation: 1, brightness: 0.8)], startPoint: UnitPoint.topLeading, endPoint: UnitPoint.bottomTrailing))
+                        Spacer()
+                            .frame(width: 20)
+                        HalfAppBannerView(app: appsVM.apps[1], gradient: LinearGradient(colors: [Color(hue: 270/360, saturation: 1, brightness: 0.8), Color(hue: 290/360, saturation: 1, brightness: 0.8)], startPoint: UnitPoint.topLeading, endPoint: UnitPoint.bottomTrailing))
+                    }
+                    Divider()
+                        .padding(.vertical)
+                    HStack {
+                        Text("Recently Opened Apps")
+                            .font(.title2)
+                            .fontWeight(.semibold)
+                        Spacer()
+                        Text("See All")
+                            .foregroundColor(.cyan)
+                    }
+                    HStack {
+                        VStack {
+                            SmallAppBannerView(app: appsVM.apps[0])
+                            SmallAppBannerView(app: appsVM.apps[0])
+                            SmallAppBannerView(app: appsVM.apps[0])
+                            SmallAppBannerView(app: appsVM.apps[0])
+                            SmallAppBannerView(app: appsVM.apps[0])
+                            SmallAppBannerView(app: appsVM.apps[0])
+                        }
+                        Spacer()
+                            .frame(width: 40)
+                        VStack {
+                            SmallAppBannerView(app: appsVM.apps[1])
+                            SmallAppBannerView(app: appsVM.apps[1])
+                            SmallAppBannerView(app: appsVM.apps[1])
+                            SmallAppBannerView(app: appsVM.apps[1])
+                            SmallAppBannerView(app: appsVM.apps[1])
+                            SmallAppBannerView(app: appsVM.apps[1])
+                        }
+                    }
+                }
+                .padding(.all)
+            }
+            .navigationTitle("Home")
+            ToastView()
```

Comment:
It's better to use `.overlay` instead of `ZStack` and putting new view.

But i think using `ViewBuilder`, `ViewModifier` and extending View is much better.